streams/src/main/java/org/apache/kafka/streams/kstream/internals/GroupedStreamAggregateBuilder.java

```diff
@@ -74,14 +75,18 @@
 
```

Comment:
Not your code, but did we want the assert on L73 to execute all the time, rather than just when java is executed with `-ea`?

Reply:
I'm not sure.  \cc @guozhangwang